src/WCSimPhysicsListFactory.cc

```diff
@@ -103,6 +107,22 @@ void WCSimPhysicsListFactory::ConstructProcess() {
         G4cout << "Enabling RadCapture nCapture process" << G4endl;
         theCaptureProcess->RegisterMe(theNeutronRadCapture);
     }
+    // Add the scintillation process
```

Comment:
Scintillation process is already defined under `WCSimOpticalPhysics` class

Reply:
I see. Honestly I do not know how to implement it from the `WCSimOpticalPhysics` class. With this code I have I see the Scintillation but if I remove it I don't.

Reply:
Inside `WCSimOpticalPhysics::ConstructProcess()` there are codes about scintillation process and particle to be applied. Your codes here are duplicated and will probably override the setup in `WCSimOpticalPhysics`.

Can you double-check whether the scintillation process is available without your modifications? I have tried some scintillator simulation before and was able to see scintillation photon generation.

Reply:
I presume @kmtsui has had a chat with you about this. But now that #393 is merged, you should be able to control the scintallation process from `jobOptions.mac`. It'd be a good idea to pull from WCSim/WCSim:develop & check this